swan-lake/development-tutorials/java-interoperability/call-java-code-from-ballerina.md

```diff
@@ -1,31 +1,40 @@
 ---
 layout: ballerina-calling-java-code-left-nav-pages-swanlake
 title: Call Java code from Ballerina
-description: See how Ballerina offers a straightforward way to call existing Java code from Ballerina.
+description: Ballerina offers a straightforward way to call the existing Java code from Ballerina. Although Ballerina is not designed to be a JVM language, the current implementation, which targets the JVM, aka jBallerina, provides Java interoperability by adhering to the Ballerina language semantics.
```

Comment:
Do we need such lenghty thing for description ?

Reply:
Fixed in https://github.com/ballerina-platform/ballerina-dev-website/pull/7839.